### DISCUSSION TITLE
UI Shell question

### DISCUSSION THREAD

**Discussion by wtrudell360** [carbon-design-system/carbon#10531] [Help] [answered]
Is there any action that allows for closing the right panel by clicking outside of the panel? IT doesn't appear to have any functionality that would enable this behavior and would require custom code to trigger closing by clicking outside of the panel.

**Comment by jnm2377** [comment, marked as answer]
Hey @wtrudell360 you're correct. We don't currently have a configuration for closing the right panel on outside click. You would have to implement that with custom logic. 👍🏽
